blockchain/tx_pool_test.go

```diff
@@ -1997,14 +2061,40 @@ func TestDynamicFeeTransactionNotAcceptedNotEnableHardfork(t *testing.T) {
 	}
 }
 
-// TestDynamicFeeTransactionAccepted tests that pool accept the transaction which has gasFeeCap bigger than or equal to baseFee.
-func TestDynamicFeeTransactionAccepted(t *testing.T) {
+func TestDynamicFeeTransactionAcceptedEip1559(t *testing.T) {
 	t.Parallel()
 	baseFee := big.NewInt(30)
 
 	pool, key := setupTxPoolWithConfig(eip1559Config)
 	defer pool.Stop()
-	pool.SetGasPrice(baseFee)
+	pool.SetBaseFee(baseFee)
```

Comment:
If this test case is for EIP-1559, don't  we need to call pool.SetGasPrice()?

Reply:
That's better, thanks!